### ISSUE TITLE
IRC Channels that start with a hash (#) need to be in quotes in the config file

### ISSUE THREAD

**Issue by maxwedwards** [gkrizek/ghi#19]
Could file this one under user error as a quote makes a comment in the yaml file but it caught me out. Perhaps the documentation and the example config files could include channel names that start with a hash and are in quotes. This would give an example and maybe stop someone from having to check the logs to see why it can't talk to their IRC channel.

**Comment by laanwj** [comment]
Yes, I've stumbled on this as well. It's an unavoidable artifact of YAML syntax but it would be good to document it as a potential footgun.
